meinberlin/apps/budgeting/assets/VoteButton.jsx

```diff
@@ -1,64 +1,64 @@
 import React from 'react'
 import django from 'django'
-import { updateItem } from '../../contrib/assets/helpers.js'
+import { useApi } from '../../contrib/assets/ApiProvider'
 
-export default class VoteButton extends React.Component {
-  async addVote () {
+export const VoteButton = props => {
```

Comment:
it looks like a lot of changes, actually just converting from class notation to function notation.